content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -392,8 +390,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
   which allows a cluster operator to live migrate
   controllers from the kube-controller-manager into an external controller-manager
   (e.g. the cloud-controller-manager) in an HA cluster without downtime.
-- `CPUManager`: Enable container level CPU affinity support, see
-  [CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/).
```

Comment:
Leave this here, unless all components no longer even recognize that feature gate name on the command line.

Reply:
I'm deleting because I'm not aware of any other component besides the kubelet that consider this feature gate.